src/nrc/api/admin.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib import admin
+from django.utils.html import format_html
+from django.utils.translation import gettext_lazy as _
+
+from vng_api_common.models import JWTSecret
+from zds_client import ClientAuth
+
+admin.site.unregister(JWTSecret)
+
+
+@admin.register(JWTSecret)
+class JWTSecretAdmin(admin.ModelAdmin):
+
+    list_display = (
+        "identifier",
+        "get_jwt",
+    )
+
+    @admin.display(description="jwt")
+    def get_jwt(self, obj):
+        if obj.identifier and obj.secret:
+            auth = ClientAuth(obj.identifier, obj.secret)
+            jwt = auth.credentials()["Authorization"]
+            return format_html(
+                '<code class="copy-action jwt" data-copy-value="{val}">{val}</code><p>{hint}</p>',
```

Comment:
`jwt` is a user-defined css class in Open Zaak, which is used to add `word-wrap: break-word`. Could you also add `break-word` here? 
And I'm not sure of showing jwt in the list view. Should we just show it in the detail view as a read-only field? What do you think?

Reply:
I would have also put it the detail view but I couldn't find out how to do easily.